pkg/asset/ignition/machine/node.go

```diff
@@ -34,7 +34,9 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 	case ovirttypes.Name:
 		ignitionHost = fmt.Sprintf("%s:22623", installConfig.Ovirt.APIVIP)
 	case vspheretypes.Name:
-		ignitionHost = fmt.Sprintf("%s:22623", installConfig.VSphere.APIVIP)
+		if installConfig.VSphere.APIVIP != "" {
+			ignitionHost = fmt.Sprintf("%s:22623", installConfig.VSphere.APIVIP)
+		}
```

Comment:
```
else {
ignitionHost = fmt.Sprintf("api-int.%s:22623", installConfig.ClusterDomain())
}

Reply:
How about `fallthrough`?

Reply:
Opted against `fallthrough` and set default to reduce duplicating code with more `else`.

Reply:
If VIP is an IP, need to future-proof for IPV6? net.JoinHostPort() per #2916

Reply:
Rebased and updated.